onnx_chainer/context.py

```diff
@@ -0,0 +1,18 @@
+class Context(object):
+
+    def __init__(self, model):
+        self.namedparams = dict()
```

Comment:
How about removing `self.namedparams` and just use `self.name_list` even for parameters?

Reply:
It sounds reasonable. I would change the code.